spec/db/models.rb

```diff
@@ -27,6 +27,9 @@ class Author < VirtualTotalTestBase
            :class_name => "Book", :foreign_key => "author_id"
 
   virtual_total :total_recently_published_books, :recently_published_books
+  virtual_average :average_recently_published_books_rating, :recently_published_books, :rating
+  virtual_minimum :minimum_recently_published_books_rating, :recently_published_books, :rating
+  virtual_maximum :maximum_recently_published_books_rating, :recently_published_books, :rating
   virtual_aggregate :sum_recently_published_books_rating, :recently_published_books, :sum, :rating
```

Comment:
since we are updating a major version for this, lets deprecate `virtual_aggregate` calls